protocol/consistencychecks.go

```diff
@@ -275,34 +292,15 @@ func (cc *ConsistencyChecks) verifyFulfilledPromise(uname string,
 //
 // 	If the request is a key lookup, and
 // 	- the request is successful, then the directory should return a promise for the lookup binding.
-func (cc *ConsistencyChecks) verifyReturnedPromise(requestType int,
-	responseCode ErrorCode, df *DirectoryProof,
+// These above checks should be performed before calling this method.
+func (cc *ConsistencyChecks) verifyReturnedPromise(df *DirectoryProof,
```

Comment:
Update the comment: https://github.com/coniks-sys/coniks-go/blob/cgo-client/protocol/consistencychecks.go#L286 states that `verifyReturnedPromise()` operates based on the request type, which no longer is true.